diff --git a/waywall/server/xwm.c b/waywall/server/xwm.c
index 4c45f37..fa9d7dd 100644
--- a/waywall/server/xwm.c
+++ b/waywall/server/xwm.c
@@ -615,7 +615,7 @@ handle_xcb_create_notify(struct xwm *xwm, xcb_create_notify_event_t *event) {
         ww_log(LOG_WARN,
                "X11 client attempted to create window (%" PRIu32 ") with override redirect",
                (uint32_t)event->window);
-        xcb_kill_client(xwm->conn, event->window);
+        // xcb_kill_client(xwm->conn, event->window);
         return;
     }
 
